src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlotting.R

```diff
@@ -32,7 +32,9 @@ create_acnv_plots_file = function(sample_name, snp_counts_file, coverage_file, s
 	segments = read.table(segments_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
     snp_counts[,"CONTIG"] = convert_XY_to_23_24(snp_counts[,"CONTIG"])
     coverage = read.table(coverage_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
-    names(coverage)[names(coverage) == sample_name] = "VALUE"
+
```

Comment:
Does that work in all cases?  I thought you were going to do a set difference on the column names.

Let's discuss in person.